libs/application-generic/src/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -66,11 +61,25 @@ export class TriggerEvent {
         actor: this.mapActor(command.actor),
       };
 
-      Logger.debug(mappedCommand.actor);
-
       const { environmentId, identifier, organizationId, userId } =
         mappedCommand;
 
+      const environment = await this.environmentRepository.findOne({
+        _id: environmentId,
+      });
+
+      if (!environment) {
+        throw new ApiException('Environment not found');
+      }
+
+      this.logger.assign({
+        transactionId: mappedCommand.transactionId,
+        environmentId: mappedCommand.environmentId,
+        organizationId: mappedCommand.organizationId,
+      });
```

Comment:
i know that we were doing in here and it is irrelevant to this PR but can we initialize it in WorkflowWorker instead?